opencti-platform/opencti-front/src/private/components/techniques/attack_patterns/AttackPatternEditionOverview.jsx

```diff
@@ -195,7 +196,7 @@ const AttackPatternEditionOverviewComponent = (props) => {
         isValid,
         dirty,
       }) => (
-        <Form style={{ marginTop: theme.spacing(2) }}>
+        <Form style={{ margin: '20px 0 20px 0' }}>
```

Comment:
Why remove useTheme() ? we prefere to set the `marginTop` with theme.spacing(2) insteaed of '20px 0 20px 0'

Reply:
So if I remember correctly, we are supposed to be moving away from using the useTheme() since that is being deprecated. I believe it was removed from previous iterations of this work.

Reply:
`useStyle()`  is deprecated, `useTheme()` is ok 😅